appcontainer/Dockerfile

```diff
@@ -25,7 +25,7 @@ RUN useradd --create-home --shell /bin/bash $USER && \
     # install server components
     apt-get update && \
     # install git for use when installing dependencies from VCS (https://pip.pypa.io/en/stable/topics/vcs-support/)
```

Comment:
We can remove this comment now. It was just documenting that we need Git in the appcontainer so that pip can install `eligibility-api` from version control. (https://github.com/cal-itp/benefits/commit/55d0b177b8b58ea646099172c3f530b52b98f198)